doc/onboarding.md

```diff
@@ -14,9 +14,8 @@ onboarding session.
 
 ## Fifteen minutes before the onboarding session
 
-* Prior to the onboarding session, add the new Collaborator to
-  [the Collaborators team](https://github.com/orgs/nodejs/teams/collaborators),
-  and to [the Members team](https://github.com/orgs/nodejs/teams/members) if
+* Prior to the onboarding session, add the new Collaborator to the
+  [the Collaborators team](https://github.com/orgs/nodejs/teams/collaborators), if
```

Comment:
Totally a nit: but the additional comma isn't necessary. It doesn't hurt anything either. But to reduce the diff to the bare minimum, maybe get rid of it so this line is unchanged?

Reply:
FWIW, this line would be changed either way, the last word comes from the next line on the left side of the diff. Speaking of which, I'd say it should stay on the next line to keep this one under 80 characters.

Reply:
I've removed the comma and got rid of the spare 'the', sorry I missed that!